packages/core/src/browser/resource-context-key.ts

```diff
@@ -38,7 +42,7 @@ export class ResourceContextKey {
     protected resourcePath: ContextKey<string>;
 
     @postConstruct()
-    protected init(): void {
+    protected async init(): Promise<void> {
```

Comment:
With the refactor of the backend OS access, I don't see any reason for this to be `async` anymore?